src/client/components/chartAndButtons/ChartAndButtons.jsx

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+import TimePeriodButtons from '../timePeriodButtons';
+import Chart from '../chart';
+
+const ChartAndButtons = () => (
```

Comment:
name isn't my favourite but I don't know what else to call it...
TimePeriodChart?
TimeSelectionChart?
SelectableTimeChart?